catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -342,6 +344,7 @@ public void destroy() {
     config.put("isExperimental", isExperimental);
     config.put("autoMergeTime", autoMergeTime);
     config.put("mapHome", mapHome);
+    config.put("product", uiName);
```

Comment:
:question: This property seems to be called `uiName` everywhere else; why is it called "`product`" in the configuration?

Reply:
The front end uses the name `product`; this was easier than refactoring every place `product` is used.